algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/models.py

```diff
@@ -258,6 +259,7 @@ def forward(self, x: Tensor) -> Tensor:
     x = self.conv1(x)
     x = self.bn1(x)
     x = self.relu(x)
+    x = F.pad(x, [0, 2, 0, 2], "constant", float("-inf"))
```

Comment:
Actually, i think that it can either be [0, 1, 0, 1] or [0, 2, 0, 2]: i was playing around with some non-default arguments of max_pool and seem to have left this one behind! I will reconfirm this and make it [0, 1, 0, 1].

Reply:
One question: were you able to run the code with the change? It seems like in cuda 11.7, I get an autograd error with this code enabled. I am not sure if this is a version-specific CUDA error.